src/dispute-logic.js

```diff
@@ -64,10 +64,29 @@ export function useDisputeLogic(disputeId) {
 
   return {
     actions,
-    dispute,
+    dispute: processDisputeData(dispute),
     disputeFetching: fetching,
     requestMode,
     panelState,
     requests,
   }
 }
+
+export function processDisputeData(dispute) {
+  if (!dispute?.metadata) {
+    return { ...dispute }
+  }
+  try {
+    const parsedDisputeData = JSON.parse(dispute.metadata)
+    // If the description tag is not present in metadata we assume that the metadata itself is the description
+    return {
+      ...dispute,
+      description: parsedDisputeData.description || dispute.metadata,
```

Comment:
I think this can just be `|| ''`; you can't use `JSON.parse()` on just a string.